content/en/docs/setup/install/operator/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Operator Install [Experimental]
+description: Install and configure Istio using the Istio Operator.
+weight: 25
+keywords: [operator,kubernetes,helm]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Follow this guide to install and configure an Istio mesh using an alternate
+installation method: the Istio {{<gloss operator>}}Operator{{</gloss>}}
+installation.
+
+The Istio Operator installation is a shorter process with the option of
+installing Istio using a one-line command. It has user input
+validation to help prevent installation errors and customization options to
+override any aspect of the configuration.
+
+It has the following additional characteristics:
+
+- Descriptive error messages to spot configuration errors easier.
+
+- Installer capable of waiting for Istio pods and services to be in a
+  ready state.
+
+- Fewer dependencies (doesn’t require Helm).
+
+The Operator install is accessed via [`istioctl`](/docs/reference/commands/istioctl/)
```

Comment:
I believe that stuff in [] doesn't need specific backticks, but I could be wrong.  Can you test?

Reply:
Ticks are required to satisfy the spell checker when istioctl is not in the dictionary, but the link and font renders correctly using this syntax. Could instead add istioctl to the spell checker dictionary but that's what I had before and changed it based on feedback.